saleor/graphql/product/tests/mutations/test_product_variant_update.py

```diff
@@ -347,15 +350,16 @@ def test_update_product_variant_skip_updating_fields_when_unchanged(
     field, value = field_values
     variables[field] = value
 
+    # when
     response = staff_api_client.post_graphql(
         QUERY_UPDATE_VARIANT_CHANGING_FIELDS,
         variables,
         permissions=[permission_manage_products],
     )
+
+    # then
     variant.refresh_from_db()
     get_graphql_content(response)
-    flush_post_commit_hooks()
```

Comment:
Should you use `django_capture_on_commit_callbacks` here or `flush_post_commit_hooks` is unnecessary?

Reply:
unnecessary, it was a leftover